lesson_7/src/app/pages/Users.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import UsersList from '../components/UsersList';
+
+export default class Users extends Component {
+  render() {
+    document.title = 'Пользователи';
+
+    return (
+      <div>
+        {
+          (!this.props.children) ?
+            (<UsersList />)
+            :
+            (this.props.children)
+        }
+      </div>
+    );
+  }
+  componentWillMount() {
```

Comment:
```suggestion
  componentDidUpdate() {
```
теперь корректно будет отображаться на вкладке